core/src/scheduler/resource/Resource.cpp

```diff
@@ -153,7 +153,7 @@ Resource::pick_task_execute() {
 
 void
 Resource::loader_function() {
-    SetThreadName("taskloader_th");
+    SetThreadName("task_loader_th");
```

Comment:
TOO LONG. The thread name is a meaningful C language string, whose length is restricted to 16 characters, including the terminating null byte ('\0').
http://man7.org/linux/man-pages/man3/pthread_setname_np.3.html

Reply:
rollback